src/panels/profile/ha-pick-language-row.js

```diff
@@ -95,15 +95,17 @@ class HaPickLanguageRow extends LocalizeMixin(EventsMixin(PolymerElement)) {
   languageSelectionChanged(newVal) {
     // Only fire event if language was changed. This prevents select updates when
     // responding to hass changes.
-    if (newVal !== this.hass.language) {
-      this.fire("hass-language-select", { language: newVal });
+    if (newVal !== this.hass.locale.language) {
+      this.fire("hass-language-select", {
```

Comment:
Should this now be `hass-locale-select`?

Reply:
I kept it as-is for now because I created a separate event name for selecting the `number_format` that does not reload translations.